.ko.yaml

```diff
@@ -1,2 +1,2 @@
 baseImageOverrides:
-  github.com/vdemeester/kobw: docker.io/vdemeester/kobw-base
+  github.com/vdemeester/kobw: gcr.io/knative-releases/github.com/knative/build/build-base:latest
```

Comment:
I assume this will be from a more official are in the future

Reply:
@arilivigni well, this one is really just for dev purpose (aka to increment quickly). The CI will publish images `vdemeester/kobw-builder` (for now) that are on top of `centos` (a bit bigger)